cli/deployments/tekton.go

```diff
@@ -61,11 +64,26 @@ func (k Tekton) Describe() string {
 		tektonPipelineReleaseYamlPath, tektonDashboardYamlPath, tektonTriggersReleaseYamlPath)
 }
 
+// Delete removes Tekton from kubernetes cluster
 func (k Tekton) Delete(c kubernetes.Cluster) error {
-	return c.Kubectl.CoreV1().Namespaces().Delete(context.Background(), tektonDeploymentID, metav1.DeleteOptions{})
+	message := "Deleting Tekton"
+	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond)
+	s.Suffix = emoji.Sprintf(" %s :zzz:", message)
+	s.Start()
+
+	err := c.Kubectl.CoreV1().Namespaces().Delete(context.Background(), tektonDeploymentID, metav1.DeleteOptions{})
+	if err != nil {
+		return errors.New("Failed uninstalling Quarks: " + err.Error())
+	}
+	s.Stop()
```

Comment:
Move up to `s.Start()` and use `defer` to ensure execution regardless of code path.

Reply:
Done with commit 84de7f0c9c6094ee525ad5b8033ae42f11225d03.